Sources/Interaction/Manipulators/MouseRangeManipulator/index.js

```diff
@@ -37,8 +37,15 @@ function vtkMouseRangeManipulator(publicAPI, model) {
     if (value !== oldValue) {
       listener.setValue(value);
       model.incrementalDelta[listener] = 0;
-    } else if (value !== listener.min && value !== listener.max) {
+    } else {
       model.incrementalDelta[listener] += delta;
+      // Do not allow incremental delta to go past range
+      if (
+        (value === listener.min && model.incrementalDelta[listener] < 0) ||
+        (value === listener.max && model.incrementalDelta[listener] > 0)
+      ) {
```

Comment:
Does it need to be that strict ? `===` vs `<` ?

Reply:
could be `<=` to be safe, but not `<` (which should never happen since value is clamped between min and max): this is to check if we are AT (===) an extremity (min or max), if we handle incrementing the delta or not:
* if at min and decreasing delta, ignore delta.
* if at min and increasing delta, increment delta.
* if at max and increasing delta, ignore delta.
* if at max and decreasing delta, increment delta.

Reply:
Sure I just wanted to make sure that equality has a chance to happen.